packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -86,6 +88,18 @@ def initialize(
         )
 
 
+def validate_display_name(display_name: str) -> None:
```

Comment:
Now that there is an initialize method, there is no reason for these validate functions to be public. 
1. Rename to `_validate_display_name` and `_validate_default_value_type`
2. Add calls to both functions in `initialize`
3. Remove calls to both functions from `measurement/service.py`
4. Retrofit tests to call underscored versions